src/plone/volto/browser/migrate_to_volto.py

```diff
@@ -105,7 +105,7 @@ def do_migrate_folders(self):
         catalog = getToolByName(self.context, "portal_catalog")
         for brain in catalog(portal_type="Folder", sort_on="path"):
             obj = brain.getObject()
-            obj = make_document(obj, slate=self.slate)
+            obj = make_document(obj, service_url=self.service_url, slate=self.slate)
```

Comment:
@pbauer just out of curiosity. What is service_url here?

Reply:
Today I released this in `plone.volto 4.0.8`, which is in Plone 6.0.3rc1.